fiftyone/factory/repos/delegated_operation.py

```diff
@@ -281,7 +281,7 @@ def list_operations(
         if paging.limit:
             docs = docs.limit(paging.limit)
 
-        registry = OperatorRegistry(enabled="all")
+        registry = OperatorRegistry()
```

Comment:
this is gone on develop and I don't think you should filter here as I think you should be able to see/list previous runs of disabled operators